src/client/renderingengine.cpp

```diff
@@ -1013,6 +1013,30 @@ const char *RenderingEngine::getVideoDriverFriendlyName(irr::video::E_DRIVER_TYP
 	return driver_names[type];
 }
 
+bool RenderingEngine::take_screenshot(const std::string &filename)
+{
+	video::IVideoDriver *driver = get_video_driver();
+	irr::video::IImage *const raw_image = driver->createScreenShot();
+
+	if (!raw_image)
+		return false;
+
+	u32 quality = (u32)g_settings->getS32("screenshot_quality");
+	quality = MYMIN(MYMAX(quality, 0), 100) / 100.0 * 255;
```

Comment:
`rangelim(quality, 0, 100)`

At least this variable is now unused in `Client::makeScreenshot`